modules/new_profile/php/new_profile.class.inc

```diff
@@ -207,127 +203,9 @@ class New_Profile extends \NDB_Form
             'edc'       => $edc,
             'sex'       => $sex,
             'pscidSet'  => $pscidSet,
-            'site'      => $site,
+            'site'      => $sites,
             'project'   => $project,
         );
-
-        $this->form->addFormRule(array(&$this, '_validateNewCandidate'));
-    }
-
-    /**
-     * Validate function
-     *
-     * @param array $values the value
-     *
-     * @return array $errors
-     */
-    function _validateNewCandidate($values): array
```

Comment:
why is the validation function being deleted ?

Reply:
Oh, I wrongly understood your request early.

Reply:
reverted